lib/events/s3_importer.rb

```diff
@@ -0,0 +1,53 @@
+module Events
+  class S3Importer
+    def initialize(s3_key)
+      @s3_key = s3_key
+    end
+
+    def import
+      file = Zlib::GzipReader.new(object.get.body)
+      csv = CSV.new(file, headers: true)
```

Comment:
If I remember right `.parse` will only work with a string and not an instance of GzipReader, will check though